src/App.tsx

```diff
@@ -40,7 +49,7 @@ function App() {
 
   switch (currentPage) {
     case 'login':
-      return <Login />;
+      return <Login onSubmit={navigateToQueuePreview} />;
```

Comment:
How about this function name ?

```suggestion
      return <Login onSubmit={handleOnLoginSubmit} />;
```